### PR TITLE
Solidity Feedback part 1

### PR DESCRIPTION
**Pull request by dvargasfuertes**
Addresses the first 11 bullets under Action Items in https://app.clarity.so/cabin/work/555

### CODE REVIEW

File: packages/app/pages/checkout/[network]/[address].tsx [hunk 1]
```diff
@@ -1,8 +1,5 @@
 import { GetServerSideProps } from "next";
```
Comment (dvargasfuertes) [inline packages/app/pages/checkout/[network]/[address].tsx]: I would click "Hide Whitespace" while reviewing this PR

File: packages/app/pages/checkout/[network]/[address].tsx [hunk 2]
```diff
@@ -209,7 +209,8 @@ const CheckoutPage = ({
           })
           .on("receipt", (receipt) => {
             const id =
-              (receipt.events?.["Purchase"]?.returnValues?.id as string) || "";
+              (receipt.events?.["Transfer"]?.returnValues?.tokenId as string) ||
```
Comment (dvargasfuertes) [inline packages/app/pages/checkout/[network]/[address].tsx]: This is the only actual change in this file, the rest is just `prettier` doing its thing

Comment (ssaitejar96) [reply packages/app/pages/checkout/[network]/[address].tsx]: Got it!

**Review by ssaitejar96 — DISMISSED** [review]
LGTM!

**Pull request merged**